### PR TITLE
Pass SkipCRDs to helmclient

### PR DESCRIPTION
**Pull request by yasn77**
Towards giantswarm/giantswarm#13746

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Pass SkipCRDs to Helm client
```
Comment (rossf7) [inline CHANGELOG.md:10]:
```suggestion
- Pass SkipCRDs to Helm client.
```

File: pkg/helmclient/install.go [hunk 2]
```diff
@@ -67,4 +67,5 @@ func (options InstallOptions) configure(action *action.Install, namespace string
 	action.ReleaseName = options.ReleaseName
 	action.Timeout = options.Timeout
 	action.Wait = options.Wait
+	action.SkipCRDs = options.SkipCRDs
```
Comment (rossf7) [inline pkg/helmclient/install.go:70]: Nice but the flag also needs to be set in update.go.

Comment (yasn77) [reply pkg/helmclient/install.go:70]: Not sure why? Since we don't use install flag (`-i`), there isn't a need for skip CRDS.

Comment (rossf7) [reply pkg/helmclient/install.go:70]: Right, good catch. 👍

**Review by rossf7 — APPROVED** [review]
LGTM

**Pull request merged**